### PR TITLE
Update Question/Quiz component design

### LINKED ISSUE #2083: Improve consistency of question/quiz component design
### Please confirm that you have searched existing issues in the repo

Yes, I have searched the existing issues

### Any related issues?

_No response_

### What is the area that this feature belongs to?

Reader Usability

### Is your feature request related to a problem? Please describe.

The question/quiz components have a card like design, which looks somewhat... out of place

But this is subjective. Feel free to input and close this issue if you don't think so.

I think some issues are:
- Large animations in quiz should be avoided.
- Box shadow is too strong
- question text weight and and size are a tad too big.

### Describe the solution you'd like

Any tweaks to make it look more consistent.

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by itsyme**
**What is the purpose of this pull request?**

- [ ] Documentation update
- [ ] Bug fix
- [x] Feature addition or enhancement
- [ ] Code maintenance
- [ ] DevOps
- [ ] Improve developer experience
- [ ] Others, please explain:

**Overview of changes:**
Resolves #2083. Changed the animation for wrong answers in Question component to a less vigorous animation. Removed box shadows for both Question and Quiz to be consistent with other components.

**Anything you'd like to highlight/discuss:**

**Testing instructions:**

**Proposed commit message: (wrap lines at 72 characters)**
Update Question/Quiz design

---

**Checklist:** :ballot_box_with_check:

- [ ] Updated the documentation for feature additions and enhancements
- [ ] Added tests for bug fixes or features
- [x] Linked all related issues
- [x] No unrelated changes

### CODE REVIEW

**Review by ong6 — COMMENTED** [review]
Looks good and addresses all of @ang-zeyu's concerns, I'll tag him here so he can have the final say for this PR

File: packages/vue-components/src/questions/Question.vue
Comment (ang-zeyu) [inline packages/vue-components/src/questions/Question.vue:399]: > Large animations in quiz should be avoided.

Just in case, I actually meant the sliding animation from question to question. 

But if this felt like an annoyance as well I'm ok with changing this too.

> sliding animation from question to question. 

Does this look OK? Personally, it was a little disorienting. I'm fine with keeping it as well otherwise.

Comment (ang-zeyu) [reply packages/vue-components/src/questions/Question.vue:399]: If you have any other ideas you'd like to add in, please do! You can get input on the design from the others here.

**Review by ang-zeyu — APPROVED** [review]

File: packages/vue-components/src/questions/Question.vue
Comment (itsyme) [reply packages/vue-components/src/questions/Question.vue:399]: I initially felt that the sliding animation was not too disorienting, however after taking another look I think there may be better alternatives we can explore. Especially for users who may want to complete a quiz quickly, I feel that this animation may be a tad too long. I will explore alternatives and create a shorter animation and push it in the next commit!

Comment (itsyme) [reply packages/vue-components/src/questions/Question.vue:399]: In the latest commit I reduced the translateX to make the transition less distracting. However, I felt that using translateY would make for a nicer effect but doing this would block the Quiz's progress bar for a split second. Do you think that would be an issue?

Comment (ang-zeyu) [reply packages/vue-components/src/questions/Question.vue:399]: > reduced the translateX

This looks much better!

> translateY

Do you have a gif or similar? Just imagining it, I don't think it'd be an issue.

Comment (itsyme) [reply packages/vue-components/src/questions/Question.vue:399]: ezgif com-optimize
Here it is! There seems to be a split second of the progress bar being blocked by the incoming question as it comes from the top.

Comment (ang-zeyu) [reply packages/vue-components/src/questions/Question.vue:399]: That looks even better than horizontal transitions.

>  split second of the progress bar being blocked by the incoming question as it comes from the top.

Is that not fixable with z-index, or did doing so create a different UX?

Comment (ang-zeyu) [reply packages/vue-components/src/questions/Question.vue:399]: Fyi: transforms create their own stacking context if its the issue https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context

Comment (itsyme) [reply packages/vue-components/src/questions/Question.vue:399]: I managed to fix the covering of the progress bar! I realised I was changing the z-index in the wrong field. I have updated the transition to translateY in the latest commit!

Comment (ang-zeyu) [reply packages/vue-components/src/questions/Question.vue:399]: Thanks, the transition looks good.

The z-index implementation can be improved slightly. If `1000` is larger than the zindex of other elements in the root/parent stacking context(s), it will appear above them. E.g. 

(I changed 1000 to 100000 to demonstrate easily here)
image

Let's create our own stacking context to isolate the z-index's impact within the `<quiz>`.

We could likely use a smaller, more controlled value (e.g. just `z-index: 1`) for easier tracking then as well. (optional)

Comment (itsyme) [reply packages/vue-components/src/questions/Question.vue:399]: Thank you for your suggestion! I agree as well and have isolated the z-index's impact within `<quiz>` and changed the `z-index` to `1` in the latest commit!

**Review by ang-zeyu — APPROVED** [review]

**Pull request merged**
